apps/guardian-ui/src/components/SetupProgress.tsx

```diff
@@ -78,25 +79,27 @@ const Step: React.FC<StepProps> = ({ text, state, isFirst }) => {
           </Center>
         </Flex>
       </Flex>
-      <Text
-        position='absolute'
-        bottom={0}
-        right={0}
-        transform={`translateX(50%) translateX(${
-          state === StepState.Active ? '-18px' : '-12px'
-        })`}
-        textAlign='center'
-        fontWeight='600'
-        fontSize={{ base: '12px', md: '14px' }}
-        color={colorState(
-          theme.colors.gray[700],
-          theme.colors.blue[600],
-          theme.colors.gray[700]
-        )}
-        whiteSpace='nowrap'
-      >
-        {text}
-      </Text>
+      {(state === StepState.Active || !isMobile) && (
+        <Text
+          position='absolute'
+          bottom={0}
+          right={0}
+          transform={`translateX(50%) translateX(${
+            state === StepState.Active ? '-18px' : '-12px'
+          })`}
+          textAlign='center'
+          fontWeight='600'
+          fontSize={{ base: '12px', md: '14px' }}
+          color={colorState(
+            theme.colors.gray[700],
+            theme.colors.blue[600],
+            theme.colors.gray[700]
+          )}
+          whiteSpace='nowrap'
+        >
+          {text}
+        </Text>
+      )}
```

Comment:
Use https://chakra-ui.com/docs/components/show-hide instead.

Reply:
fixed in following commit